src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages_es.properties

```diff
@@ -38,4 +39,4 @@ summary.javaDoc=Resumen fragmento Prohibida.
 summary.javaDoc.missing=Resumen javadoc falta.
 tag.continuation.indent=Línea continuación tiene nivel de sangría incorrecta, nivel esperado debería ser {0}
 type.missingTag=Al comentario Javadoc le falta una etiqueta {0}.
-type.tagFormat=El comentario Javadoc {0} debe coincidir con el patrón ''{1}''.
+type.tagFormat=El etiqueta Javadoc {0} debe coincidir con el patrón ''{1}''.
```

Comment:
This doesn't seem to be related. Why is this being changed?

Reply:
comentario = comment
etiqueta = tag
and it should be tag as mentioned in message.properties

Reply:
This should then be split this into it's own separate PR since its unrelated to the main issue. It should use the `spelling` prefix.